packages/SwingSet/src/kernel/vatManager/manager-local.js

```diff
@@ -110,6 +111,7 @@ export function makeLocalVatManagerFactory(tools) {
       vatParameters,
       virtualObjectCacheSize,
       gcTools,
+      marshalSaveError,
```

Comment:
There's something funny going on here. `managerOptions` now accepts `liveSlotsConsole` and `marshalSaveError`, ok good. When building a local manager, `marshalSaveError` defaults to the pre-existing `vatConsole`, also good. Liveslots gets a new `liveSlotsConsole` argument, which defaults to `console`, good.

But:
* this call to `makeLiveSlots` is passing in `marshalSaveError` to the argument that liveslots names `console` or `liveSlotsConsole`, instead of passing in `liveSlotsConsole`
* liveslots gives its `liveSlotsConsole` to marshal as the argument it knows as `marshalSaveError`
* liveslots console messages (via `lsdebug()` calls, or manually-added `console.log` calls) go to whatver `console` global endowment it received at eval time, which for local vats is the `kernelEndowment` console
  * it looks like the `liveSlotsConsole` argument is meant to serve that purpose instead

This seems a bit confused.

Maybe liveslots should accept a `marshalSaveError` argument instead of one named `liveSlotsConsole`? Or maybe it should accept both, with a caveat that `liveSlotsConsole` is not currently used (and maybe in the future it will be used for `lsdebug` or something?

Or maybe the vatmanager factory should accept `liveSlotsConsole` and *not* `marshalSaveError`, liveslots should only accept `liveSlotsConsole`, and we have the caveat that `liveSlotsConsole` is only currently used for `marshalSaveError`?

Reply:
The general approach looks good to me, but I think these mismatched names are going to be confusing.

Reply:
Thanks for noticing this.  I got confused.  `marshalSaveError` should not be used except as an option in the `makeMarshal` call.  I will remove it from the `managerOptions` (and use `liveSlotsConsole`).